config/locales/client.en.yml

```diff
@@ -571,7 +571,7 @@ en:
       topic_stat_sentence:
         one: "%{count} new topic in the past %{unit}."
         other: "%{count} new topics in the past %{unit}."
-      more: "more"
+      more: " (%{count} more) ..."
```

Comment:
@vinothkannans You should always use pluralization (`one` and `other`) when you add a translation with `%{count}` in it. The need for an empty space at the beginning of a text is also a bad sign. See my above comment regarding concatenating strings.

Reply:
@gschlager sure, I will do the necessary changes here.